src/wasm-lib/kcl/src/std/appearance.rs

```diff
@@ -283,17 +289,18 @@ async fn inner_appearance(data: AppearanceData, solid_set: SolidSet, args: Args)
             a: 100.0,
         };
 
-        args.batch_modeling_cmd(
+        batch_cmd!(
```

Comment:
so instead of doing this, sorry it might be cleaner and less prone to errors if this lived in engine/mod.rs, and then each implementation kept track of it, then that way we could clear it when we clear all the other shit thru engine

Reply:
because i fear doing it on each send, is going to make it so someone forgets one time versus it just magically happening in engine/mod.rs

Reply:
it would be super similar to how we keep track of batches already https://github.com/KittyCAD/modeling-app/blob/main/src/wasm-lib/kcl/src/engine/conn.rs#L44 but its own data struct thru there

Reply:
also not saying you have to go this route my only fear was someone forgetting the macro if that would lead to a compile time error im fine with this as is

Reply:
Yes. All those `Arc<Mutex<_>>`s in `EngineConnection` feel so gross to me, so I was trying to use what is actually mutable through a `&mut`. But you're right. We want to make the API so that it can't be used incorrectly. So it's worth it.